python-sdk/nuscenes/eval/common/loaders.py

```diff
@@ -2,30 +2,32 @@
 # Code written by Oscar Beijbom, 2019.
 
 import json
-from typing import Dict, Tuple
+from typing import Dict, List, Tuple
 
 import numpy as np
 import tqdm
-from pyquaternion import Quaternion
-
 from nuscenes import NuScenes
 from nuscenes.eval.common.data_classes import EvalBoxes
 from nuscenes.eval.detection.data_classes import DetectionBox
 from nuscenes.eval.detection.utils import category_to_detection_name
 from nuscenes.eval.tracking.data_classes import TrackingBox
 from nuscenes.utils.data_classes import Box
 from nuscenes.utils.geometry_utils import points_in_box
-from nuscenes.utils.splits import create_splits_scenes
+from nuscenes.utils.splits import get_scenes_of_split
+from pyquaternion import Quaternion
 
 
-def load_prediction(result_path: str, max_boxes_per_sample: int, box_cls, verbose: bool = False) \
+def load_prediction(result_path: str, max_boxes_per_sample: int, box_cls, verbose: bool = False,
```

Comment:
Could we leave `load_prediction` and `load_gt` untouched, and instead create standalone functions that will load predictions and GTs based on user-specified sample tokens? E.g. `load_prediction_of_sample_tokens` and `load_gt_of_sample_tokens` (or some other better naming :smile:)

Reply:
Sure, I'll implement it soon

Reply:
Just implemented this in the latest commit [Keep original load_prediction and load_gt](https://github.com/nutonomy/nuscenes-devkit/pull/1046/commits/1cc1ebbd25ca7f43143e003c01379dc95fa35510). 

Could you give me a quick feedback if this is how you imagine the `load_*` function being called from the tracking and detection `evaluate.py` files?

Then, I would go on tidying up and testing more. This includes e.g.:
- a clear-cut line of where it makes sense to support and don't support custom splits. E.g. do we need to offer `load_prediction_of_sample_tokens` besides `load_prediction` in `generate_panoptic_labels`?

Reply:
@michael-hoss your PR is looking nice! I agree with the overall concept you have implemented (we can go over the details of the code once you have tidied and clean things up more).

As for where to support and not support custom splits, I think supporting custom splits for detection and tracking will suffice for this PR